xray/src/generation.rs

```diff
@@ -180,16 +180,8 @@ impl ColoringStrategy for XRayColoringStrategy {
         discretized_locations: Vec<Point3<u32>>,
     ) {
         for d_loc in discretized_locations {
-            match self.z_buckets.entry((d_loc.x, d_loc.y)) {
-                Entry::Occupied(mut e) => {
-                    e.get_mut().insert(d_loc.z);
-                }
-                Entry::Vacant(v) => {
-                    let mut set = FnvHashSet::default();
-                    set.insert(d_loc.z);
-                    v.insert(set);
-                }
-            }
+            let z_buckets = self.z_buckets.entry((d_loc.x, d_loc.y)).or_default();
+            z_buckets.insert(d_loc.z);
```

Comment:
nit: use one-liner?
```suggestion
            self.z_buckets.entry((d_loc.x, d_loc.y)).or_default().insert(d_loc.z);
```

Reply:
Tried this before, cargo fmt doesn't make it a one-liner, but a three- or four-liner :(

Reply:
:+1: